.github/workflows/main.yml

```diff
@@ -95,19 +97,51 @@ jobs:
           rm codeql-home/codeql.zip
         shell: bash
 
-      - name: Run unit tests (Linux)
+      - name: Get Nightly Release URL
+        id: get-url
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+        shell: bash
+        # This workflow step gets an unstable testing version of the CodeQL CLI. It should not be used outside of these tests.
+        run: |
+          export LATEST=`gh release list --repo dsp-testing/codeql-cli-nightlies -L 1 | cut -f 3`
```

Comment:
Minor: How confident do we feel about relying on the format of the output here?  If we're not sure that this is stable, we could instead use something like `gh api repos/dsp-testing/codeql-cli-nightlies/releases --jq '.[].tag_name' --method GET --raw-field 'per_page=1'`.

Reply:
Good idea, that seems safer.